### ISSUE TITLE
This is really good project but...

### ISSUE THREAD

**Issue by lessload** [cbeuw/Cloak#169]
This is really good project but something dev should consider.
- name "cloak" is hard to find topic relate in search engine like google. Make this project hard to get information and unpopular. recommend to change the name to something like "ck-cloak"
- re-design logo. (just recommend not importance much)
- wiki too low information for newbie. just open for community to edit it. something like https://github.com/DNSCrypt/dnscrypt-proxy/wiki do.

This project really good.
In my opinion concept of this project better than "stunnel". but need a lot of feature develop and security audit.

**Comment by notsure2** [comment]
Yes it has a big advantage of not needing any TLS certificate on the server side.
